blazar_dashboard/content/leases/tables.py

```diff
@@ -114,6 +123,9 @@ class LeasesTable(tables.DataTable):
     class Meta(object):
         name = "leases"
         verbose_name = _("Leases")
-        table_actions = (ViewLeaseCalendar, ViewNetworkReservationCalendar,
-                         CreateLease, DeleteLease, )
+        tactions = [ViewLeaseCalendar, ViewNetworkReservationCalendar,
+                    CreateLease, DeleteLease]
+        if conf.device_reservation.get('enabled'):
```

Comment:
Nice, I was wondering how we were going to do this. Also, for some reason I didn't think you could put a conditional in a class body like this, 🤯